stories/layout/Table.stories.js

```diff
@@ -45,6 +45,14 @@ export const WithSuppliedData = withInfo()(() => (
   </Panel>
 ));
 
+export const ResponsiveTable = withInfo()(() => (
+  <Box maxWidth="1100">
+    <Panel>
+      <Table p="200" data={data} freezeFirstColumn freezeColumns={2} />
```

Comment:
Not seeing freezeColumns in the props of Table, is this left over from something else?

Reply:
Oh yep, was experimenting